migration/src/m0000610_improve_version_cmp_fns/mavenver_cmp.sql

```diff
@@ -0,0 +1,110 @@
+
+create or replace function mavenver_cmp(left_p text, right_p text)
```

Comment:
I guess we have to do this logic in the db? Too much potentially returned to post-process? All the SQL makes my eyes bleed on to the vomit I just threw up. :eyes: :fork_and_knife: :vomiting_face:

Reply:
Yep, we want to do it in the DB, so that we can also support suitable pagination.

Else, we have to bring them *all* back, and scrobble around on the rust side winnowing things out of the version ranges.

Reply:
Add just a touch of Sriracha, and the bloodvomit will be lovely.

Reply:
I was once in a band named "bloodvomit"